packages/text-field/index.tsx

```diff
@@ -320,32 +308,25 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
       textarea,
     } = this.props;
     const {foundation} = this.state;
-    const textField = (
-      <div
-        {...this.otherProps}
-        className={this.classes}
-        onClick={() => foundation && foundation.handleTextFieldInteraction()}
-        onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
-        key='text-field-container'
-      >
-        {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
-        {foundation ? this.renderInput() : null}
-        {label && !outlined && !fullWidth ? this.renderLabel() : null}
-        {outlined ? this.renderNotchedOutline() : null}
-        {!fullWidth && !textarea && !outlined ? this.renderLineRipple() : null}
-        {trailingIcon ? this.renderIcon(trailingIcon, onTrailingIconSelect) : null}
-      </div>
-    );
 
-    if (helperText) {
-      return (
-        <React.Fragment>
-          {textField}
-          {this.renderHelperText()}
-        </React.Fragment>
-      );
-    }
-    return textField;
+    return (
+      <React.Fragment>
+        <div
+          {...this.otherProps}
+          className={this.classes}
+          onClick={() => foundation!.handleTextFieldInteraction()}
+          onKeyDown={() => foundation!.handleTextFieldInteraction()}
+          key='text-field-container'>
+          {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
+          {this.renderInput()}
+          {label && !fullWidth ? this.renderLabel() : null}
```

Comment:
The label should be rendered inside the notched outline if its `outlined`. See https://github.com/material-components/material-components-web-react/pull/816

Reply:
Reflect it!

Also I change adding outlined class name when text-field is not fullWidth.
(https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield#full-width)
Confirm this please :)

Reply:
looks good!